tests/_testutils.py

```diff
@@ -9,28 +11,12 @@ def noop(*args, **kwargs):
     return None
 
 
-class MockRuntimeStorage:
-    def __init__(self, key_prefixes):
-        self._cache = {}
-
-    def load(self, key, of_type=None, default=None):
-        """Unstable API."""
-        if key in self._cache:
-            value = self._cache[key]
-        else:
-            value = None
-
-        if value is None:
-            return deepcopy(default)
-        elif of_type and not isinstance(value, of_type):
-            return deepcopy(default)
-        else:
-            return deepcopy(value)
-
-    def store(self, key, value):
-        """Unstable API."""
-        self._cache[key] = value
-        return value
+class MockRuntimeStorage(RuntimeStorage):
+    def __init__(self, key_prefixes, backend=None):
+        if not backend:
+            run_dir = mkdtemp('run_dir')
```

Comment:
should this use the pytest tempdir utility?

Reply:
I would have preferred to use it, yes.  But I couldn't figure out how to use the magic `tempdir` fixture within an arbitrary class.  Do you know how it can be done?

Reply:
I suppose it could be passed in, but that is a but gross.